chopsticks-sample/src/main/resources/dev/chopsticks/sample/app/dstreams/dstreams-sample-worker-app.conf

```diff
@@ -0,0 +1,23 @@
+akka {
+  coordinated-shutdown.run-by-jvm-shutdown-hook = on
+
+  http.server.preview.enable-http2 = on
+
+  log-dead-letters = off
+  loglevel = INFO
+
+  stream.materializer.debug.fuzzing-mode = off
```

Comment:
This should already be `off` by default and it's only set to `on` under test `resources`